Formula/lua@5.3.rb

```diff
@@ -90,6 +90,9 @@ def install
   end
 
   def pc_file
+    libs = %w[-llua.#{version.major_minor} -lm]
```

Comment:
```suggestion
    libs = %W[-llua.#{version.major_minor} -lm]
```

Reply:
Thanks!  I'm also pretty sure now that it should be `-llua#{version.major.minor}` and not `-llua.#{version.major.minor}`, because the library is `liblua5.3.so`, not `liblua.5.3.so`.  I'll push all these changes and then it should be ready.